traffic_ops/testing/api/v1/user_test.go

```diff
@@ -296,8 +296,8 @@ func GetTestUsers(t *testing.T) {
 
 func GetTestAdminUsers(t *testing.T) {
 	adminUsers, _, err := TOSession.GetUsersByRole("admin")
-	if err != nil {
-		t.Errorf("cannot GET users by role: %v", err)
+	if err != nil || len(adminUsers) == 0 || adminUsers[0].RoleName == nil {
```

Comment:
If this test errors because `adminUsers` is `nil`/empty or its first member's `RoleName` property is `nil` then it'll print `cannot GET admin users by role: <nil>` which isn't super helpful. But what's worse is that in that case the test continues because it's only an error, not a fatal one. So it would still segfault.

Reply:
gotcha. still learning a lot about these tests